### DISCUSSION TITLE
File preview in macOS doesnt work properly since Sequoia

### DISCUSSION THREAD

**Discussion by Senajs** [mikekazakov/nimble-commander#441] [Q&A]
when i select a file and tap "space" usually a preview of the file comes. Works fine in Finder but in NC sometimes i just see a folder symbol and when i tap again the filename enters edit mode . i then have to escape and do it again
As far as i remember this started with Sequoia and was working fine before.. or am i missing something?

**Comment by mikekazakov** [comment]
Not exactly sure what's going on here...
Can you provide screenshots or video records of what is happening?
Preview of a folder is normally just a folder icon, nothing more - that's expected.
By tapping you mean mouse clicks on a filename or something else?

I'm not aware of any changes in Sequoia that changes this behaviour.

**Reply by Senajs** [reply]
ill try to repro and record - i was also surprised..
